spectator-agent/src/main/resources/bitbucket.conf

```diff
@@ -0,0 +1,132 @@
+// http://confluence.atlassian.com/bitbucketserver/enabling-jmx-counters-for-performance-monitoring-776640189.html
+netflix.spectator.agent.jmx {
+  mappings = ${?netflix.spectator.agent.jmx.mappings} [
+    {
+      query = "com.atlassian.bitbucket.thread-pools:name=*"
+      measurements = [
+        {
+          name = "bitbucket.{name}.MaximumPoolSize"
+          value = "{MaximumPoolSize}"
+          tags = [ ]
```

Comment:
This should set `atlas.dstype=gauge`. See example:

https://github.com/Netflix/spectator/blob/master/spectator-agent/src/main/resources/cassandra.conf#L1533

And for background:

https://github.com/Netflix/atlas/wiki/Concepts#normalization